### PR TITLE
Strategy symbol optional

### PR DESCRIPTION
**Pull request by ChaituVR**
Fixes https://github.com/snapshot-labs/snapshot/issues/1020

### CODE REVIEW

File: src/strategies/erc721/schema.json [hunk 1]
```diff
@@ -10,7 +10,6 @@
           "type": "string",
           "title": "Symbol",
           "examples": ["e.g. DOODLE"],
-          "minLength": 1,
```
Comment (samuveth) [inline src/strategies/erc721/schema.json:13]: Shouldn't symbol also be removed inside required?
image

**Comment by samuveth** [comment]
@bonustrack should we make symbol optional on all strategies?

File: src/strategies/erc721/schema.json (hunk 1 above)
Comment (ChaituVR) [reply src/strategies/erc721/schema.json:13]: Removed 👍

**Comment by samuveth** [comment]
@ChaituVR I don't think you need to remove the minLength if you remove the required.

**Comment by ChaituVR** [comment]
@samuveth If we leave minLength and If we pass an empty string it throws an error,
image


But why do we need minLength now anyway? any problem by removing it? 😅

**Comment by bonustrack** [comment]
> @bonustrack should we make symbol optional on all strategies?

@samuveth On all strategies yes

**Comment by bonustrack** [comment]
@ChaituVR Was with with removing symbol in "required"?

**Comment by samuveth** [comment]
> @samuveth If we leave minLength and If we pass an empty string it throws an error, image
> 
> But why do we need minLength now anyway? any problem by removing it? 😅

Not sure how we are handling this one the backend but if we don't have a minlength the symbol could be an empty string.

**Comment by samuveth** [comment]
> > But why do we need minLength now anyway? any problem by removing it? 😅
> 
> Not sure how we are handling this one the backend but if we don't have a minlength the symbol could be an empty string.

**Comment by samuveth** [comment]
> @samuveth On all strategies yes

So yeah @ChaituVR all strategies need this change.

**Comment by ChaituVR** [comment]
> On all strategies yes

@bonustrack Does it also mean in the future, anyone wants to add a strategy with "symbol" required, we should change it to optional? I mean is it always optional regardless of strategy/ its owner?

**Comment by bonustrack** [comment]
@ChaituVR Yes, ideally we would move at some point the symbol param into the parent object like network example:
```json
{
  "name": "ticket",
  "symbol": "ABC",
  "network": "1",
  "params": {}
}
```
This way we wont need to force ourself some schema for strategy params

**Comment by ChaituVR** [comment]
Awesome @bonustrack we should do that soon 👍 made it optional in all strategies and added a test case so users will not make it required for future strategies

**Comment by samuveth** [comment]
> Awesome @bonustrack we should do that soon 👍 made it optional in all strategies and added a test case so users will not make it required for future strategies

Would be good to maybe create an issue for it

**Review by samuveth — APPROVED** [review]

**Pull request merged**